src/services/tunables/tunables_service.cpp

```diff
@@ -48,9 +54,12 @@ namespace big
 			if (!m_script_started)
 			{
 				SCRIPT::REQUEST_SCRIPT_WITH_NAME_HASH("tuneables_processing"_J);
+				SCRIPT::REQUEST_SCRIPT_WITH_NAME_HASH("tunables_registration"_J);
 
-				if (SCRIPT::HAS_SCRIPT_WITH_NAME_HASH_LOADED("tuneables_processing"_J))
+				if (SCRIPT::HAS_SCRIPT_WITH_NAME_HASH_LOADED("tuneables_processing"_J) && SCRIPT::HAS_SCRIPT_WITH_NAME_HASH_LOADED("tunables_registration"_J))
 				{
+					m_num_tunables = gta_util::find_script_program("tunables_registration"_J)->m_global_count - 0x40000;
```

Comment:
i logged this value `m_num_tunables` and also the `m_tunables` size after the processing ended. They did not match (35973 & 34421). Is it fine?

Reply:
According to my decompile, there are 34625 instances of a write to a global from tunables_processing. There seems to be an error with the m_tunables generator. m_num_tunables is probably the total size of memory allocated into the Globals space for tunables.